src/con_duct/__main__.py

```diff
@@ -15,6 +16,11 @@
 from typing import IO, Any, Optional, TextIO
 from . import __version__
 
+lgr = logging.getLogger("con-duct")
+# Default log level is DEBUG, but can be overridden by setting DUCT_LOG_LEVEL
+# and later set via CLI argument
+DEFAULT_LOG_LEVEL = os.environ.get("DUCT_LOG_LEVEL", "DEBUG").upper()
```

Comment:
```suggestion
DEFAULT_LOG_LEVEL = os.environ.get("DUCT_LOG_LEVEL", "INFO").upper()
```
DEBUG is too low for default IMO

Reply:
Right!